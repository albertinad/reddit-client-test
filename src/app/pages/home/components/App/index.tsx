import React from 'react';

const App: React.FunctionComponent = () => (
  <span>Hello Reddit Client</span>
);

export default App;
