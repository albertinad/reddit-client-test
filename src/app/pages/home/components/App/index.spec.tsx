import React from 'react';
import { render } from '@testing-library/react';
import App from './index';

describe('App dummy Component', () => {
  it('should render', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
