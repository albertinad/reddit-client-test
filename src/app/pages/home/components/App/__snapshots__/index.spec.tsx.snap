// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App dummy Component should render 1`] = `
<div>
  <span>
    Hello Reddit Client
  </span>
</div>
`;
